frontend/public/components/default-resource.tsx

```diff
@@ -130,9 +188,22 @@ const TableRowForKind: React.FC<RowFunctionArgs<K8sResourceKind>> = ({ obj, cust
           t('public~None')
         )}
       </TableData>
-      <TableData className={tableColumnClasses[2]}>
-        <Timestamp timestamp={obj.metadata.creationTimestamp} />
-      </TableData>
+      {!checkColumnsForCreationTimestamp(additionalPrinterColumns) && (
```

Comment:
Another option would be to just not ever show creation timestamp if there are custom printer columns, which would bring us closer to the CLI behavior.